src/js/services/ng-scores.js

```diff
@@ -112,23 +112,13 @@ define('services/ng-scores',[
                 self.beginupdate();
                 try {
                     // Determine scores file version
-                    var scores;
-                    var version;
-                    if (res.version === undefined) {
-                        // 'Legacy' storage, all scores stored directly
-                        // as an array
-                        scores = res;
-                        version = 1;
-                    } else {
-                        // New style storage, scores in a property,
-                        // and an explicit version identifier.
-                        version = res.version;
-                        scores = res.scores;
```

Comment:
Why was the old version removed? It is useful to allow old data to be loaded for testing purposes.
Seems it should be fairly easy to keep this backwards compatibility, too?

Reply:
Because version 3 uses id, not index. Working with index can easily cause errors, and I thought it would be best to not support it, by that forcing the users to use id oriented scores

Reply:
Ah. I think it would be better to have the server do a one-time version-upgrade then, e.g. when it starts, such that it can add (random) ids to each entry.

Reply:
Will do